code-postprocessing/cocopp/testbedsettings.py

```diff
@@ -59,6 +60,8 @@
     default_suite_bi: default_testbed_bi,
     'bbob-biobj-ext': default_testbed_bi_ext,
     'bbob-constrained': default_testbed_cons,
+    'bbob-constrained-active-only': default_testbed_cons,
```

Comment:
this looks strange, should these be here at all?

Reply:
Yes, I guess  the `bbob-constrained` is the only one to be officially released